ui/desktop/electron-app/src/services/client-agent-daemon-manager.js

```diff
@@ -15,7 +15,11 @@ class ClientAgentDaemonManager {
    */
   async status() {
     const clientAgentStatusCommand = ['client-agent', 'status', '-format=json'];
-    const { stdout, stderr } = spawnSync(clientAgentStatusCommand);
+    // The status command can take a while to execute if it's not running as it
+    // retries multiple times so we should set a timeout to not wait too long
```

Comment:
Note that this may change in the future as backend will likely remove the retry ([slack thread](https://hashicorp.slack.com/archives/C05EHAH84HE/p1723836029423339))